apps/frontend/src/pages/projects/components/project-create-form/project-create-form.tsx

```diff
@@ -45,6 +55,17 @@ const ProjectCreateForm = ({ onSubmit }: Properties): JSX.Element => {
 				name="description"
 				rowsCount={4}
 			/>
+			{charCount <= ProjectValidationRule.DESCRIPTION_MAXIMUM_LENGTH && (
```

Comment:
create a boolean variable for this
```suggestion
			{isDescriptionCounterShown
```